### PR TITLE
5 build change password page

### LINKED ISSUE #5: Build Change Password Page

### PR DESCRIPTION
**Pull request by dagaither**
## What type of PR is this? (check all applicable)

- [x] Feature
- [x] Refactor
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Other (please specify here: <type here>)
  
  
## Description of what this Pull Request does
Login.jsx: Added link to ForgotPassword.jsx
Register.jsx: Disable submit button if passwords do not match.
ChangePassword.jsx: Created
ForgotPassword.jsx: Created
router.jsx: Updated with new routes/elements

## Related Tickets & Documents
- Closes #5

### CODE REVIEW

File: frontend/platoon-console/src/pages/ChangePassword.jsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    const handleSubmit = async (e) => {
+        e.preventDefault();
+        setSuccessMessage("");
+        setErrorMessage("");
+        try {
+            const userData = {
+                "new_password": newPassword
+            };
+
+            let response = await axios.post(`http://127.0.0.1:8000/user/password-reset/${uuid64}/${token}/`, userData, {
```
Comment (L-Carr) [inline frontend/platoon-console/src/pages/ChangePassword.jsx:29]: Change to put request.  => old : axios.post =>: new: axios.put

**Review by L-Carr — CHANGES_REQUESTED** [review]
The url link that takes the uuid and the token. Make that a put request instead of post

**Review by L-Carr — APPROVED** [review]
Looks good!

**Review by DeeJaeMann — APPROVED** [review]
Looks Good

**Pull request merged**